public/contribute.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "Ensemble",
+    "description": "Ensemble is the platform that powers the Firefox Public Data Report (https://data.firefox.com), a weekly public report on the activity, behavior, and hardware configuration of Firefox Desktop users.",
```

Comment:
Do you want to remove "Desktop" here and be vague? Or we can leave it in, and maybe remove it in the future (memories permitting) when we add Mobile metrics?

Reply:
+1 for vague. Good catch!